src/schemes/boundary/open_boundary/system.jl

```diff
@@ -484,3 +484,4 @@ end
 
 @inline viscosity_model(system::OpenBoundarySPHSystem, neighbor_system::FluidSystem) = neighbor_system.viscosity
 @inline viscosity_model(system::OpenBoundarySPHSystem, neighbor_system::BoundarySystem) = neighbor_system.boundary_model.viscosity
+@inline viscosity_model(system, neighbor_system::OpenBoundarySPHSystem) = system.viscosity
```

Comment:
```suggestion
# When available, use the viscosity of the neighbor system in the interaction.
# So, e.g. for fluid-boundary interaction, we use the boundary's viscosity, which can be used
# to model no-slip boundary conditions.
@inline viscosity_model(system::OpenBoundarySPHSystem, neighbor_system::FluidSystem) = neighbor_system.viscosity
@inline viscosity_model(system::OpenBoundarySPHSystem, neighbor_system::BoundarySystem) = neighbor_system.boundary_model.viscosity
@inline viscosity_model(system, neighbor_system::OpenBoundarySPHSystem) = system.viscosity
```
From the perspective of a new user, it would be great if you could add this comment here and in all other locations where you define this function.
Or would it make sense to define all of them together somewhere?

Reply:
Only if we want to move these functions to viscosity.jl.

Reply:
Is this possible or do we not have access to the system types there?

Reply:
ah yes that would be an issue with the **OpenBoundarySPHSystem**